### PR TITLE
[chore] Update approver auto assign config

### PR DESCRIPTION
**Pull request by bryan-aguilar**
**Description:** Add missing approvers to the auto assign config. I also reorganized the maintainers list to match the ordering of names in the README. Small nit, but it made it easier to compare.

### CODE REVIEW

**Review by dashpole — APPROVED** [review]

**Review by crobert-1 — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -11,19 +11,21 @@ useAssigneeGroups: true
...
+    - astencel-sumo
     - Aneurysm9
     - atoulme
     - dashpole
+    - songy23
     - fatsheep9146
     # Maintainers
-    - bogdandrutu
     - codeboten
+    - bogdandrutu
```
Comment (codeboten) [inline .github/auto_assign.yml:22]:
```suggestion
    - bogdandrutu
    - codeboten
```

File: .github/auto_assign.yml [hunk 2]
```diff
@@ -11,19 +11,21 @@ useAssigneeGroups: true
 assigneeGroups:
   approvers_maintainers:
     # Approvers
+    - astencel-sumo
     - Aneurysm9
```
Comment (codeboten) [inline .github/auto_assign.yml:15]:
```suggestion
    - Aneurysm9
    - astencel-sumo
```

File: .github/auto_assign.yml [hunk 3]
```diff
@@ -11,19 +11,21 @@ useAssigneeGroups: true
...
-    - bogdandrutu
     - codeboten
+    - bogdandrutu
     - djaglowski
     - dmitryax
     - evan-bradley
     - jpkrohling
-    - MovieStoreGuy
     - mx-psi
+    - MovieStoreGuy
```
Comment (codeboten) [inline .github/auto_assign.yml:28]:
```suggestion
    - MovieStoreGuy
    - mx-psi
```

**Review by atoulme — APPROVED** [review]

**Review by codeboten — COMMENTED** [review]
sorting seemed incorrect in a few cases

**Comment by crobert-1** [comment]
> sorting seemed incorrect in a few cases

I believe sorting was done on the people's names, rather than username.

**Comment by codeboten** [comment]
@crobert-1 yeah i saw this after making the suggestions... i found it more confusing than useful tbh

**Pull request merged**